es5.js

```diff
@@ -230,6 +255,10 @@ module.exports = {
     'no-useless-call': 2,
     // disallow unnecessary concatenation of literals or template literals
     'no-useless-concat': 2,
+    // disallow unnecessary escape usage
+    'no-useless-escape': 2,
+    // disallow redundant return statements
+    'no-useless-return': 2,
```

Comment:
Just noting that since we enabled this one, it is possible to run into eslint/eslint#7626